### ISSUE TITLE
Interleukin terms

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7298]
**Preferred term label:**
Various new terms related to supplement of
https://pubmed.ncbi.nlm.nih.gov/32238915/

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
